ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphUtils.ts

```diff
@@ -0,0 +1,64 @@
+import { Model, NodeModel } from '@patternfly/react-topology';
```

Comment:
should we move the other `utils.tsx` file to this directory as well and rename it to be more specific like `modelUtils.tsx` or something?

Reply:
Yea we can do that